### ISSUE TITLE
website down?

### ISSUE THREAD

**Issue by annaritz** [BaderLab/openPIP#83]
Hi - your software looks great, and we'd like to take a look at your docker container. However, there seems to be an issue with the website that's listed in your paper: http://openpip.baderlab.org/

Any help is appreciated. Thanks!

**Comment by annaritz** [comment]
It looks like your README.md contains a link to the description (http://baderlab.org/Software/openPIP) but the live demo still seems broken. Thanks.

**Comment by Moemen-Gaballah** [comment]
> It looks like your README.md contains a link to the description (http://baderlab.org/Software/openPIP) but the live demo still seems broken. Thanks.

Hello, Dear annaritz 
[Demo](http://openpip.baderlab.org/) is down

**Comment by MoHelmy** [comment]
Yes, I know the demo is down. There are 2 other live instances that I linked above, both are live.
